
Turn 5
================================================================
warning: entity 2 was directed to use 19 halite to move north, but only 0 halite was available
At command 1 of 3:
>>> 1   m 2 n
    2   m 0 o
    3   g


Turn 6
================================================================
error: failed to decode bot message: "T" (at position 1)
Last input received was:
Traceback (most recent call last):
  File "MyBot.py", line 167, in <module>
    ships.remove(me.get_ship(id))
ValueError: list.remove(x): x not in list

